$NetBSD$

configure: You requested to build `web2c' using an installed `kpathsea' version,
configure:     which requires to locate the <kpathsea/paths.h> header file.
configure: error: Sorry, not found under any of: /usr/include /usr/local/include *****

--- configure.orig	2013-04-08 16:36:05.000000000 +0900
+++ configure	2013-11-02 17:54:15.000000000 +0900
@@ -20056,7 +20056,7 @@ fi
 
 if test "x$enable_web2c" = xyes && test "x$with_system_kpathsea" = xyes; then
   if test "x$with_kpathsea_includes" = x; then
-    list="/usr/include /usr/local/include"
+    list="@@PREFIX@@/include /usr/include /usr/local/include"
   else
     list=$with_kpathsea_includes
   fi
